Makefile

```diff
@@ -9,10 +9,8 @@ PROJ = google_containers
 
 PKG_SOURCES := $(shell find pkg -name '*.go')
 
-node-problem-detector: ./bin/node-problem-detector
-
-./bin/node-problem-detector: $(PKG_SOURCES) node_problem_detector.go
-	CGO_ENABLED=0 GOOS=linux godep go build -a -installsuffix cgo -ldflags '-w' -o node-problem-detector
+node-problem-detector: $(PKG_SOURCES) node_problem_detector.go
+	GOOS=linux go build -ldflags '-w -extldflags "-static"' -o node-problem-detector
```

Comment:
@Random-Liu According to code in https://github.com/kubernetes/node-problem-detector/blob/master/pkg/kernelmonitor/kernel_log_watcher.go#L83-L103. I do not think make `logPath` to empty in `kernel-monitor.json` will make node-problem-detector use systemd. It will uses default `/var/log/kern.log`.